website/blog/2024-06-24-release-1.11.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Podman Desktop 1.11 Release
+description: Podman Desktop 1.11 has been released!
+slug: podman-desktop-release-1.11
+authors: cdrage
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.11/light-mode.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.11 Release! 🎉
+
+![Podman-desktop-1-11-hero](img/podman-desktop-release-1.11/banner.png)
+
+This release introduces:
+
+- **Light mode!**: Our most-requested feature is here! Try out our new experimental light mode in the settings.
+- **macOS Rosetta support for Apple Silicon**: Build AMD64 binaries at near-native speeds as ARM64 builds.
+- **Kubernetes improvements**: Check out our new Node and Volume pages for Kubernetes.
+- **Improved UI**: In addition to our light mode, you'll notice updates to our container listing page.
+- **Increased manifest support**: You'll now see images grouped together if built with a manifest.
```

Comment:
'You'll now see images grouped together if built with a manifest.' -> simpler 'Images built with a manifest are now grouped together'?